app/src/panels/meter/preview.svg

```diff
@@ -0,0 +1,29 @@
+<svg width="156" height="96" viewBox="0 0 156 96" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_1280_146)">
+        <g filter="url(#filter0_d_1280_146)">
+            <rect x="18" y="10" width="120" height="76" rx="6" fill="var(--card-face-color)" class="glow" />
+            <rect x="19" y="11" width="118" height="74" rx="5" stroke="var(--theme--primary)"
+                stroke-width="2" />
+        </g>
+        <rect x="66" y="64" width="24" height="4" rx="1" fill="var(--theme--primary-subdued)" />
+        <rect x="57" y="71" width="41" height="4" rx="1" fill="var(--theme--primary-subdued)" />
+        <path
+            d="M118 64C118 58.7471 116.965 53.5457 114.955 48.6927C112.945 43.8396 109.999 39.4301 106.284 35.7157C102.57 32.0014 98.1604 29.055 93.3073 27.0448C88.4543 25.0346 83.2529 24 78 24C72.7471 24 67.5457 25.0346 62.6927 27.0448C57.8396 29.055 53.4301 32.0014 49.7157 35.7157C46.0014 39.4301 43.055 43.8396 41.0448 48.6927C39.0346 53.5457 38 58.7471 38 64L54 64C54 60.8483 54.6208 57.7274 55.8269 54.8156C57.033 51.9038 58.8008 49.258 61.0294 47.0294C63.258 44.8008 65.9038 43.033 68.8156 41.8269C71.7274 40.6208 74.8483 40 78 40C81.1517 40 84.2726 40.6208 87.1844 41.8269C90.0962 43.033 92.742 44.8008 94.9706 47.0294C97.1992 49.258 98.967 51.9038 100.173 54.8156C101.379 57.7274 102 60.8483 102 64H118Z"
+            fill="var(--theme--primary-subdued)" />
+        <path
+            d="M56.2145 30.4532C50.5639 34.1227 45.9317 39.1601 42.7479 45.098C39.564 51.0358 37.9316 57.6817 38.0022 64.4189L54.0013 64.2513C53.959 60.209 54.9384 56.2215 56.8487 52.6588C58.759 49.0961 61.5383 46.0736 64.9287 43.8719L56.2145 30.4532Z"
+            fill="color-mix(in srgb, var(--theme--background), var(--theme--primary) 25%)" />
```

Comment:
@rijkvanzanten Your new theming doesn't have a 25% equivalent of primary, should I include the custom color mixin here or ...thoughts?

Reply:
This custom mixin works for me!